regulatory_compliance/containerinstance.sp

```diff
@@ -0,0 +1,32 @@
+control "container_instance_container_group_encryption_using_cmk" {
+  title       = "Azure Container Instance container group should use customer-managed key for encryption"
+  description = "Secure your containers with greater flexibility using customer-managed keys. When you specify a customer-managed key, that key is used to protect and control access to the key that encrypts your data. Using customer-managed keys provides additional capabilities to control rotation of the key encryption key or cryptographically erase data."
+  query       = query.container_instance_container_group_encryption_using_cmk
+
+  tags = merge(local.regulatory_compliance_synapse_common_tags, {
```

Comment:
This control is wrongly tagged , please create a tag for this service and tag it.